chain/network/src/peer_manager.rs

```diff
@@ -404,8 +405,8 @@ impl PeerManagerActor {
         let target_peer_id = full_peer_info.peer_info.id.clone();
 
         let new_edge = Edge::new(
-            self.peer_id.clone(),   // source
-            target_peer_id.clone(), // target
```

Comment:
Unrelated to the PR, but, rather than using comments, I'd extracted `let source` and `let = target` variables.